hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/AbstractExecutor.java

```diff
@@ -1,97 +1,275 @@
 package org.apache.mesos.hdfs.executor;
 
-import com.google.inject.Guice;
-import com.google.inject.Inject;
-import com.google.inject.Injector;
-import org.apache.commons.lang.StringUtils;
+import com.github.mustachejava.DefaultMustacheFactory;
+import com.github.mustachejava.Mustache;
+import com.github.mustachejava.MustacheFactory;
+import com.google.protobuf.InvalidProtocolBufferException;
+//import org.apache.commons.lang.StringUtils;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.apache.mesos.Executor;
 import org.apache.mesos.ExecutorDriver;
-import org.apache.mesos.MesosExecutorDriver;
-import org.apache.mesos.Protos.CommandInfo;
+//import org.apache.mesos.MesosExecutorDriver;
+//import org.apache.mesos.Protos.CommandInfo;
 import org.apache.mesos.Protos.ExecutorInfo;
 import org.apache.mesos.Protos.FrameworkInfo;
 import org.apache.mesos.Protos.SlaveInfo;
-import org.apache.mesos.Protos.Status;
+//import org.apache.mesos.Protos.Status;
+import org.apache.mesos.Protos.TaskID;
 import org.apache.mesos.Protos.TaskInfo;
 import org.apache.mesos.Protos.TaskState;
 import org.apache.mesos.Protos.TaskStatus;
-import org.apache.mesos.hdfs.config.HdfsFrameworkConfig;
-import org.apache.mesos.hdfs.file.FileUtils;
+//import org.apache.mesos.hdfs.file.FileUtils;
+import org.apache.mesos.hdfs.gen.Protos.ExecutorData;
+import org.apache.mesos.hdfs.gen.Protos.TaskData;
+import org.apache.mesos.hdfs.gen.Protos.TaskData.Parameter;
 import org.apache.mesos.hdfs.util.HDFSConstants;
 import org.apache.mesos.hdfs.util.StreamRedirect;
 
-import java.io.BufferedWriter;
-import java.io.File;
+//import java.io.BufferedWriter;
+//import java.io.File;
 import java.io.FileOutputStream;
 import java.io.IOException;
+import java.io.OutputStream;
 import java.io.OutputStreamWriter;
 import java.io.Writer;
 import java.nio.charset.Charset;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
+import java.nio.charset.StandardCharsets;
+//import java.nio.file.Files;
+//import java.nio.file.Path;
```

Comment:
A lot of commented out values here and I don't think it makes sense to rename this class.